governance.md

```diff
@@ -18,88 +36,180 @@ The committee MUST:
+
+### Election Officers
+
+For every election, the governance board wll choose three election officers, by the following criteria, so as to promote healthy rotation and diversity:
+
+- election officers must be eligible to vote
+- two election officers should have served before
+- one election officer should have never served before
+- each officer should come from a different company to maintain 40% maximal representation
+- election officers should not be currently running in the election
+
+The governing board can decide to make exceptions to the above requirements if needed (for example, if two people cannot be found who have served before and want to be officers).
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected governance board member, the candidate with the next most votes from the previous election will be offered the seat. This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will be held as soon as possible. Eligible voters from the most recent election will vote in the special election (ie: eligibility will not be redetermined at the time of the special election). A board member elected in a special election will 
+serve out the remainder of the term for the person they are replacing, regardless of the length of that remainder.
+
+### Changes to governing board
+
+When someone joins the governing board:
+
+- They should be added to [the list of current governing board members](#current-members)
+- They will be added as admins to [the CDEvents GitHub org](https://github.com/cdevents/) via the `governance` GitHub team
+- They will be added to these mailing lists as owners:
+  - [`cdevents-governance`](https://groups.google.com/g/cdevents-governance)
+  - [`cdevents-nominations`](https://groups.google.com/g/cdevents-nominations)
+  - [`cdevents-dev`](https://groups.google.com/g/cdevents-dev)
+  - [`cdevents-users`](https://groups.google.com/g/cdevents-users)
+  - [`cdevents-code-of-conduct`](https://groups.google.com/g/cdevents-code-of-conduct)
+
+When someone leaves the governing board:
+
+- They should be moved from
+  [the list of current governing board members](#current-members) to
+  [the list of former members](#former-members-%EF%B8%8F)
+- They will be removed as admins from [the CDEvents GitHub org](https://github.com/cdevents)
+- They will be removed from the `governance` GitHub team
+- They will be removed as owners from these mailing lists:
+  - [`cdevents-governance`](https://groups.google.com/g/cdevents-governance)
+  - [`tekton-nominations`](https://groups.google.com/g/cdevents-nominations)
+  - [`cdevents-dev`](https://groups.google.com/g/cdevents-dev)
+  - [`cdevents-users`](https://groups.google.com/g/cdevents-users)
+  - [`cdevents-code-of-conduct`](https://groups.google.com/g/cdevents-code-of-conduct)
+
+## Governing board leave policy
+
```

Comment:
Does it make sense to give chance to GB members to assign proxies/alternates for the entire time of their leave but capped at maximum length, e.g., 3 months?

Also, would it be possible to assign proxy/alternate for meetings individually?

Reply:
I'm not sure we need proxies for individual meetings... if a GB member cannot join an individual meeting, it's fine.
I recommend using the mailing list for decisions in any case, so they are documented.